src/utils/posthog.ts

```diff
@@ -0,0 +1,9 @@
+import { PostHog } from "posthog-node";
+
+import { env } from "~/env";
+
+const posthog = new PostHog(env.POSTHOG_API_KEY, {
+  host: "https://app.posthog.com",
+});
```

Comment:
nb. is there any way we can identify these events as coming from the Discord bot? It will make it easier to avoid mixing events from web vs. bots